src/DeepCopy/DeepCopy.php

```diff
@@ -85,11 +85,33 @@ public function copy($value)
         return $this->recursiveCopy($value);
     }
 
+    /**
+     * @param Filter  $filter
```

Comment:
I wouldn't add the phpdoc. When it doesn't provide any more information than the typehints (like here), it just adds noise IMO.

Reply:
i think you right, thanks